eastern/job_manager.py

```diff
@@ -46,6 +46,15 @@ def is_completed(self):
         status = self.kubectl.get_job_status(self.job_name)
         return (status.failed + status.succeeded > 0) and status.active == 0
 
+    def is_succeeded(self):
+        """
+        Check if job was completed and was succeeded.
+
+        :rtype: bool
+        """
+        status = self.kubectl.get_job_status(self.job_name)
+        return status.failed == 0 and status.active == 0
```

Comment:
I'm thinking of changing this to `return not status failed and not status.active`.

But it probably harm readability...